### ISSUE TITLE
Building from source ubuntu 22

### ISSUE THREAD

**Issue by realsetvin** [HTHcoin/helpthehomelesscoin#101]
Encountered many issues, is anyone able to build this from source on ubuntu 22?

**Comment by realsetvin** [comment]
$ sudo wget https://github.com/codablock/bls-signatures/archive/v20181101.zip
$ sudo unzip v20181101.zip
$ cd bls-signatures-20181101
$ sudo cmake .
$ sudo make install

This does not work, make fails. There are several errors

**Comment by realsetvin** [comment]
error: size of array element is not a multiple of its alignment 101 | blake2s_state S[8][1]; 

https://github.com/Raptor3um/raptoreum/issues/48#issuecomment-969125200

**Comment by realsetvin** [comment]
If I fix that issue there are still other issues. And thats just to build the deps, not even building the qt

**Comment by Altcoin-Cash** [comment]
I think you are the first trying on ubuntu 22.

**Comment by realsetvin** [comment]
> I think you are the first trying on ubuntu 22.

Yes sir I can only be grateful that there is someone who is willing to look at this at all. If you need anything from me let me know. 

ubuntu 24 is coming soon, and there is no binaries for any linux - so it would be nice if we could get this working.

**Comment by realsetvin** [comment]
> I think you are the first trying on ubuntu 22.

ser plz help

Is there a way I can build this in ubuntu 18 but it will run on ubuntu 22?

**Comment by nucash-mining** [comment]
It will run in Ubuntu 22

**Comment by MichaelHDesigns** [comment]
If you build for Ubuntu 18 it will run with Ubuntu 18, if you got the executables then upgrade to Ubuntu 22, they will run, but if you build from source, the depends have to be specific to your build environment. What are the errors you are getting? There are already compiled binaries available to download, but if you are building from source, what's the error?
